src/main/target/MATEKF722SE/target.h

```diff
@@ -0,0 +1,184 @@
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * INAV is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with INAV.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+#pragma once
+
+#define USE_TARGET_CONFIG
+
+#define TARGET_BOARD_IDENTIFIER "MK7S"
+#define USBD_PRODUCT_STRING  "MatekF722SE"
+
+#define LED0                    PA14  //Blue   SWCLK
+#define LED1                    PA13  //Green  SWDIO
+
+#define USE_BEEPER
+#define BEEPER_PIN              PC13
+#define BEEPER_INVERTED
+
+// *************** SPI1 Gyro & ACC *******************
+#define USE_TARGET_IMU_HARDWARE_DESCRIPTORS
+
+#define USE_SPI
+#define USE_SPI_DEVICE_1
+#define SPI1_SCK_PIN            PA5
+#define SPI1_MISO_PIN           PA6
+#define SPI1_MOSI_PIN           PA7
+
+#define USE_ACC
+#define USE_GYRO
+#define USE_DUAL_GYRO
+
+#define USE_ACC_MPU6000
+#define USE_GYRO_MPU6000
+#define MPU6000_CS_PIN          PB2
+#define MPU6000_SPI_BUS         BUS_SPI1
+#define MPU6000_EXTI_PIN        PC4
+
+#define USE_ACC_MPU6500
+#define USE_GYRO_MPU6500
+#define MPU6500_CS_PIN          PC15
+#define MPU6500_SPI_BUS         BUS_SPI1
+#define MPU6500_EXTI_PIN        PC3
+
+#define GYRO_MPU6000_ALIGN      CW180_DEG_FLIP
+#define ACC_MPU6000_ALIGN       CW180_DEG_FLIP
+
+#define GYRO_MPU6500_ALIGN      CW90_DEG
+#define ACC_MPU6500_ALIGN       CW90_DEG
+
+#define USE_EXTI
+#define USE_MPU_DATA_READY_SIGNAL
+
+// *************** I2C /Baro/Mag *********************
+#define USE_I2C
+#define USE_I2C_DEVICE_1
+#define I2C1_SCL                PB8
+#define I2C1_SDA                PB9
+
+#define USE_BARO
+#define BARO_I2C_BUS            BUS_I2C1
+#define USE_BARO_BMP280
+#define USE_BARO_MS5611
+
+#define USE_MAG
+#define MAG_I2C_BUS             BUS_I2C1
+#define USE_MAG_HMC5883
+#define USE_MAG_QMC5883
+#define USE_MAG_IST8310
+#define USE_MAG_IST8308
+#define USE_MAG_MAG3110
+#define USE_MAG_LIS3MDL
+
+// *************** SPI2 OSD ***********************
+#define USE_SPI_DEVICE_2
+#define SPI2_SCK_PIN            PB13
+#define SPI2_MISO_PIN           PB14
+#define SPI2_MOSI_PIN           PB15
+
+#define USE_OSD
+#define USE_MAX7456
+#define MAX7456_SPI_BUS         BUS_SPI2
+#define MAX7456_CS_PIN          PB12
+
+// *************** SPI3 SD BLACKBOX*******************
+#define USE_SDCARD
+#define ENABLE_BLACKBOX_LOGGING_ON_SDCARD_BY_DEFAULT
+
+#define USE_SPI_DEVICE_3
+#define SPI3_SCK_PIN            PC10
+#define SPI3_MISO_PIN           PC11
+#define SPI3_MOSI_PIN           PC12
+
+#define SDCARD_SPI_INSTANCE     SPI3
+#define SDCARD_SPI_CS_PIN       PD2
+
+#define SDCARD_DMA_CHANNEL_TX               DMA1_Stream7
```

Comment:
```suggestion
#define SDCARD_DMA_CHANNEL_TX               DMA1_Stream5
```

Reply:
It will conflict with TIM2_UP (S5/S6) if Dshot is supported.  I prefer SDCARD_DMA on DMA1_ST5